NEWS

```diff
@@ -66,6 +66,115 @@ Master (not on release branches yet)
 - Removed embedded VampirTrace.  It is in maintenance mode since 2013.
   Please consider Score-P (score-p.org) as an external replacement.
 
+2.0.0
+------
+
+ **********************************************************************
+ *  Open MPI is now fully MPI-3.1 compliant
+ **********************************************************************
+
+- Add support for UCX transport.
+- Add support for PMIx.  Version 1.1.2 of PMIx is included internally
+  in this release.
+- Add support for PLFS file systems in Open MPI I/O.
+- Update the internal copy of ROMIO to that which shipped in MPICH
+  3.1.4.
+- Update internal copy of libevent to v2.0.22.
+- Update internal copy of hwloc to v1.11.1.
+- Reduction in memory footprint for jobs with sparse communication
+  patterns at scale
+- Performance improvements to MPI RMA
+- Removed checkpoint/restart code due to loss of maintainer :-(
+- Add a new MCA parameter - opal_common_verbs_want_fork_support. This replaces
+  the btl_openib_want_fork_support parameter.
+- Add a -tune command line option to simplify setting many environment
+  variables and mca parameters.
+- Removed --enable-mpi-profiling configure option.
+- Add --with-platform-patches-dir
+- Add a new mca parameter default-dash-to-host to offer a mirror of the default_hostfile.
+- Add a  --with-pmi-libdir configure option for environments that install
+  PMI libs in a non-default location
+- Add the ability to specify the number of desired slots in the --host option.
+- Remove support for OS X Leopard
+- Remove --enable-dist configure option
```

Comment:
`--enable-dist` was always an internal optional, anyway -- probably don't need to list it here.